src/lib.rs

```diff
@@ -667,6 +667,29 @@ impl<C: Verification> Secp256k1<C> {
     /// be used for Bitcoin consensus checking since there may exist signatures
     /// which OpenSSL would verify but not libsecp256k1, or vice-versa. Requires a
     /// verify-capable context.
+    ///
+    /// ```rust
+    /// # extern crate secp256k1;
+    /// # #[cfg(feature="rand")]
+    /// # extern crate rand;
+    /// #
+    /// # fn main() {
+    /// # #[cfg(feature="rand")] {
+    /// # use rand::OsRng;
+    /// # use secp256k1::{Secp256k1, Message};
```

Comment:
You forgot to add the Error here, https://travis-ci.org/github/rust-bitcoin/rust-secp256k1/jobs/683364113#L727
```rust
    /// # use secp256k1::{Secp256k1, Message, Error};
```

Reply:
Thank you.